publish_jenkins_console.py

```diff
@@ -3,7 +3,7 @@
 import json
 import os
 import re
-from collections import OrderedDict
+from collections import defaultdict
```

Comment:
defaultdict will not keep the order but ALL_BUILDS = OrderedDict(set) did not work. I didn't want to spend much time with it because it would probably require to create custom dictionary class so I used defaultdict. I can investigate a bit more if you think OrderedDict is something we really need.

Reply:
@FilipB Yes, keeping the order is important.
What do you think about using order dict with set?

Example:
```python
ALL_BUILDS = OrderedDict()
ALL_BUILDS['job_name'] = set()
ALL_BUILDS['job_name'].add(id)
```